### ISSUE TITLE
Be more specific in how we share validation status

### ISSUE THREAD

**Issue by doobneek1** [streetlives/yourpeer.nyc#215]
I want to automate the following: we should publish who validated the location and by what means:
let's say doobneek visited the location a month ago, they check that box and input the info, the front end would indicate: d validated this location in person a month ago, if Kiesha just called, then she can check that box while re-validating and the status will update: K also called them a day ago to re-check the validity.

**Comment by doobneek1** [comment]
I think since we are sharing who we are on the about page, we can hyperlink who we are on those records for transparency,  accountability,  and interactiveness

**Comment by shakilhossain1** [comment]
Is this being done in the extension @doobneek1

**Comment by doobneek** [comment]
Yes,  @shakilhossain1. The issue is 2 years old. A lot I had to handle on my own
